### DISCUSSION TITLE
Cached/dynamic values for report fields

### DISCUSSION THREAD

**Discussion by cmahrl** [Syslifters/sysreptor#47] [General]
### Problem
Currently fields such as affected hosts have to be manually filled with every new finding. There is no way to define a dynamic list of  values for fields (e.g. for affected hosts) that could be reused in other findings.

### Solution
When inputting values to certain fields (e.g. affected hosts) , the values should be cached so that they can be reused in other Findings.

**Comment by MWedl** [comment]
Combobox fields in findings now show suggestions from other findings in the dropdown menu. Suggestions can be enabled by setting the field type to `combobox` (instead of `string`).
